mne/preprocessing/ica.py

```diff
@@ -2396,20 +2405,36 @@ def corrmap(icas, template, threshold="auto", label=None,
 
     all_maps = [_get_ica_map(ica) for ica in icas]
 
-    target = all_maps[template[0]][template[1]]
+    if len(template) == 2:
+        target = all_maps[template[0]][template[1]]
+        is_subject = True
```

Comment:
Are we trying to plot a component from one of the Ica objects (one for each subject), indicated by list indices (subject and IC)? Or are we trying to plot a bare array?
Should I add a comment?